### PR TITLE
[RHACS] Added AssumeRole integration info

### PR DESCRIPTION
**Pull request by gaurav-nelson**
- Added information about adding ECR by using AssumeRole.

Applies to 3.68

<hr>

NOTE:

- This PR is for RHACS docs.
- It doesn't require any special labels or milestones.

### CODE REVIEW

**Review by sagidlow — COMMENTED** [review]
Couple of changes needed :) Let me know if you have any questions!

File: integration/integrate-with-image-registries.adoc [hunk 1]
```diff
@@ -49,6 +49,19 @@ include::modules/manual-configuration-image-registry-ocp.adoc[leveloffset=+2]
 
 include::modules/manual-configuration-image-registry-ecr.adoc[leveloffset=+2]
 
+[id="use-assumerole-with-ecr"]
+==== Using assumerole with Amazon ECR
+
+link:https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html[AssumeRole] allows access to AWS resources without manually configuring each user's permissions.
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply integration/integrate-with-image-registries.adoc]: Thanks @sagidlow The suggestion changes the meaning, but I've re-written it based on IBM Style Guide.

File: modules/configuring-assumerole-acs.adoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+.. *Registry ID*: The ID of the registry.
+.. *Region*: The region for the registry.
+. If you have configured AssumeRole with container IAM leave the *Use container IAM role* option checked. Otherwise, uncheck the *Use custom IAM role* and enter the details for the following fields:
+.. *Access key ID*: The access key ID for the role.
+.. *Secret access key*: The secret access key for the role.
+. Select the *Use AssumeRole* checkbox.
+. Enter the details for the following fields:
+.. *AssumeRole ID*: The ID of the role to assume.
+.. *AssumeRole External ID* (optional): If you are using an link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html[external ID with AssumeRole], you can enter it here.
+. Select *Test* (*`checkmark`* icon) to test that the integration with the selected registry is working.
```
Comment (gaurav-nelson) [inline modules/configuring-assumerole-acs.adoc:28]: @sagidlow I've changed the procedure to reflect changes in the product, can you review this one again?

**Comment by gaurav-nelson** [comment]
🙇🏼 Thank you @sagidlow I've addressed your comments, can you please check and merge?

**Pull request merged**